src/DynamoApplications/StartupUtils.cs

```diff
@@ -398,45 +399,55 @@ public static List<Exception> CheckAssemblyForVersionMismatches(Assembly assembl
         private static List<Exception> GetVersionMismatchedReferencesInAppDomain(Assembly assembly, String[] assemblyNamesToIgnore)
         {
             // Get all assemblies that are currently loaded into the appdomain.
-            var loadedAssemblies = AppDomain.CurrentDomain.GetAssemblies().ToList();
             // Ignore some assemblies(Revit assemblies) that we know work and have changed their version number format or do not align
             // with semantic versioning.
+            var loadedAssemblies = AppDomain.CurrentDomain.GetAssemblies()
+                .Where(a => !assemblyNamesToIgnore.Contains(a.GetName().Name))
+                .ToList();
```

Comment:
@aparajit-pratap any idea if AppDomain.CurrentDomain.GetAssemblies() will return all assemblies from all load contexts?

Reply:
No, it's best to use `AssemblyLoadContext.All.SelectMany(context => context.Assemblies);` for that purpose.
In any case, we should avoid continuing to use `AppDomain` going forward.